thirdeye/thirdeye-pinot/src/main/java/com/linkedin/thirdeye/detector/function/AnomalyFunction.java

```diff
@@ -37,6 +49,24 @@
       throws Exception;
 
   /**
+   * Computes the score and severity according to the current and baseline of the given timeSeries and stores the
+   * information to the merged anomaly. The start and end time of the time series is provided
+   *
+   * @param anomalyToUpdated
+   *          the merged anomaly to be updated.
+   * @param timeSeries
+   *          The metric time series data.
+   * @param knownAnomalies
+   *          Any known anomalies in the time range.
+   * @return
+   *         A list of anomalies that were not previously known.
+   * @return the severity according to the current and baseline of the given timeSeries
+   */
+  void updateMergedAnomalyInfo(MergedAnomalyResultDTO anomalyToUpdated, MetricTimeSeries timeSeries,
```

Comment:
It has time keys, but not start and end of the values of current or baselines. The assumption is that the time series have all the time keys that are needed for calculating the weight of the merged anomaly.

Actually, the start and end time can be retrieved from the merged anomaly, but I want to make the signature of this function similar to that of analyze() method.